views/Geographic/Feature.svelte

```diff
@@ -1,13 +1,14 @@
 <script>
   export let color;
   export let path;
+  export let hasAnnotation = false;
 </script>
 
 <g class={color.colorClass}>
   <path
     class="q-choropleth-feature"
     fill={color.customColor && color.customColor.length > 0 ? color.customColor : 'currentColor'}
-    stroke="#fff"
-    stroke-width="0.3px"
+    stroke={hasAnnotation ? "#000000" : "#fff"}
```

Comment:
https://nzzdev.github.io/Storytelling-Styleguide/#/colors?a=gray-scale
For "black" we use `s-color-gray-9`

Reply:
Understood. But how do I set the `stroke` property to `s-color-gray-9` without just copying and pasting the hex-code?

Reply:
https://css-tricks.com/currentcolor/
There's this css-ting called `currentColor`. You can add the sophie-color as a class and then use it like this `stroke="currentColor"`

Reply:
Hm. Good idea, but this will collide with the `fill` property, which already uses the `currentColor` from the element above.